libsolidity/CompilerContext.h

```diff
@@ -94,6 +94,8 @@ class CompilerContext
 	eth::AssemblyItem appendJumpToNew() { return m_asm.appendJump().tag(); }
 	/// Appends a JUMP to a tag already on the stack
 	CompilerContext&  appendJump(eth::AssemblyItem::JumpType _jumpType = eth::AssemblyItem::JumpType::Ordinary);
+	/// Returns an "ErrorTag"
```

Comment:
hmm I'm not sure. it just returns a new created error tag. Are you sure it should be "the"?